.github/ISSUE_TEMPLATE/erddap-dataset-initial-submission.md

```diff
@@ -30,6 +30,7 @@ A more detailed written and visual description of every step is available respec
 - [ ] ERDDAP Dataset Documentation 
 - [ ] ERDDAP Test Locally 
 - [ ] Add Dataset to Development Branch
+- [ ] ERDDAP Global Attributes are matching the Metadata Record associated field
```

Comment:
Could mention the tool in the metadata form that exports globals to erddap

Reply:
Oh yeah good added. I just added a text since now that snippet is easily available on the form!!!